.cookiecutter/includes/tox/deps

```diff
@@ -0,0 +1 @@
+{dev,tests,functests,lint}: sqlalchemy>2
```

Comment:
h-testkit itself doesn't depend on sqlalchemy. This is because I imagine in future `h-testkit` might contain some non-sqlalchemy fixtures that might be used by some projects that don't depend on sqlalchemy. I don't want sqlalchemy to get added to projects as an indirect dependency of h-testkit.

But sqlalchemy does need to be installed in h-testkit's own test environments otherwise h-testkit's own tests will fail when it tries to import sqlalchemy. Hence sqlalchemy is added here as a test dependency rather than added to `setup.cfg`.